internal/builder/packages.go

```diff
@@ -25,6 +26,9 @@ type BuildOptions struct {
 
 	CreateZip   bool
 	SignPackage bool
+
+	// buildDir is the base build directory to use. Used for testing now.
+	buildDir string
```

Comment:
Similar question: why is this field required?

Reply:
To have a predictable place where packages are built for testing, without affecting the developer working copy.

Reply:
It's probably a matter preference, but I would rather keep exposed fields only in `BuildOptions` and introduce another function for testing purposes. For example:

`BuildPackage(options BuildOptions)` calls `buildPackageWithBuildDir(options BuildOptions, buildDir string)`. You can use the second one for testing.  Same story with `BuildPackagesDirectory`.

WDYT?